packages/app-store/larkcalendar/lib/CalendarService.ts

```diff
@@ -189,7 +194,9 @@ export default class LarkCalendarService implements Calendar {
   async updateEvent(uid: string, event: CalendarEvent, externalCalendarId?: string) {
     const eventId = uid;
     let eventRespData;
-    const [mainHostDestinationCalendar] = event.destinationCalendar ?? [];
+    const mainHostDestinationCalendar = event.destinationCalendar?.find(
+      (cal) => cal.externalId === externalCalendarId
+    );
```

Comment:
I see more `const [mainHostDestinationCalendar]` in this file. Should we update those as well?